buildSrc/src/main/groovy/io.micronaut.build.internal.nats-module.gradle

```diff
@@ -0,0 +1,21 @@
+plugins {
+    id 'io.micronaut.build.internal.module'
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compileOnly 'com.google.code.findbugs:jsr305' // for "warning: unknown enum constant When.MAYBE"
+
+    testRuntimeOnly 'ch.qos.logback:logback-classic:1.2.7'
+}
+
+test {
+    useJUnitPlatform()
+
+    jvmArgs '-Duser.country=US'
```

Comment:
mmm, if we don't do this already we should think about having that directly in the common plugin

Reply:
If I checked this correctly, the MicronautBuildCommonPlugin already applies those settings.
https://github.com/micronaut-projects/micronaut-build/blob/44d6162a24b98859b58440217c88d339f4a6b772/src/main/groovy/io/micronaut/build/MicronautBuildCommonPlugin.groovy#L112 

Can you double check this, @melix?

Reply:
You're correct, so the whole block can be removed